redo.dev/src/css/custom.css

```diff
@@ -56,3 +56,10 @@
 .footer {
     background-color: #1b1b1b;
 }
+.hideComponent {
```

Comment:
The only reason we have this .css file is for docusaurus components like `footer`. For components we have control over, we've just been inlining the CSS. Can we translate "hideComponent" and "demoContainer" over?